.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,6 @@
+blank_issues_enabled: true
+issue_templates:
+  - name: 🔍 Typings Inconsistency
```

Comment:
Huh, I forgot to submit this comment, sorry:

Type definition is what DefinitelyTyped calls it, so let's use that word. Maybe "Web API type definition issue". An issue would be more general than inconsistency which wouldn't cover type enhancements.